Doc/Makefile

```diff
@@ -38,6 +38,8 @@ help:
 	@echo "  serve      to serve the documentation on the localhost (8000)"
 
 build:
+	-mkdir build
```

Comment:
A more common idiom for this is `mkdir -p build`.  That has the advantage of not producing any messages if the directory already exists.